uber/configspec.ini

```diff
@@ -51,6 +51,11 @@ hostname = string(default="localhost")
 url_root = string(default="http://localhost:8282")
 url_base = string(default="%(url_root)s%(path)s")
 
+# We expose some basic services, like an attendee lookup, via WebSocket.
+# However, without a client cert configured, anyone can access these,
+# so this is off by default.
+api_enabled = boolean(default=False)
```

Comment:
Amusingly, this was the cause of our one merge conflict. Who knew that both you and I would select the same spot in configspec.ini to add something?